File: Tarea1Unidad3Java/src/Productos/Limpieza.java
package Productos;
import Productos.helpers.tipoProducto;

public class Limpieza extends Producto {
    private String marca;

    public Limpieza(String nombre, double precio, int stock, String fechaImportacion, String numSerie, String marca) {
        super(nombre,tipoProducto.LIMPIEZA, precio, stock, fechaImportacion, numSerie);
        this.marca = marca;
    }
    public String getMarca() {
        return marca;
    }

    @Override
    public String getDatos(){
        return String.format("%s    Marca: %s", super.getDatos(), getMarca());
    }
}

File: Tarea1Unidad3Python/Limpieza.py
from Producto import Producto, tipoProducto
from enum import Enum

class Limpieza(Producto):
    def __init__(self, nombre, precio, stock, fecha_importacion, marca):
        super().__init__(nombre, tipoProducto.LIMPIEZA, precio, stock, fecha_importacion)
        self.marca = marca

    def get_marca(self):
        return self.marca

    def get_datos(self):
        return f"{super().get_datos()}    Marca: {self.get_marca()}"